cypress/e2e/tables-share.cy.js

```diff
@@ -30,7 +30,7 @@ describe('Manage a table', () => {
 		cy.get('.NcTable table tr th').last().find('button').click({ force: true })
 		cy.get('.v-popper__popper.v-popper--theme-dropdown.action-item__popper.v-popper__popper--shown').contains('Share').click({ force: true })
 		cy.get('.sharing input').type(localUser2.userId)
-		cy.get('ul.multiselect__content li').first().click()
+		cy.wait(1000).get('.sharing input').type('{enter}')
```

Comment:
Maybe we can get the list entry before pressing enter to avoid timing issues. cy.wait is usually a bad idea and makes tests less stable

Reply:
I'd say merging is fine, if this turns out to make tests flaky we can investigate deeper.